zipkin-storage/cassandra3/src/test/java/zipkin/storage/cassandra3/CassandraSpanStoreTest.java

```diff
@@ -97,6 +98,17 @@ public void overFetchesToCompensateForDuplicateIndexData() {
         .hasSize(traceCount);
   }
 
+  /**
+   * Currently, trace ids are stored as a variable-length number (as opposed to a 2 part hi/lo
+   * 128bit id). In order to support query by low 64 bits, we need to split this up somehow or add a
+   * separate index.
+   */
+  @Override
+  @Test
+  @Ignore("fetch by lower 64-bit isn't supported in cassandra3")
+  public void getTrace_retrieves128bitTraceIdByLower64Bits() {
```

Comment:
thanks for opening https://github.com/openzipkin/zipkin/issues/1364
will let the discussion flow from there…